config.toml

```diff
@@ -37,7 +37,7 @@ list_pages = false
 default_theme = "dark"
 mensch_links = [
   { path = "@/e/_index.md", name = "Events", icon = "calendar" },
-  { path = "@/w/_index.md", name = "Talent", icon = "users" },
+  { path = "@/w/_index.md", name = "People", icon = "users" },
```

Comment:
Main already has this change https://github.com/tpwres/teczki-polskiego-wrestlingu/blob/1d3f3d919e3b9c98cd750bd91b3508b8a6a09717/config.toml#L40 so this is probably an effect of using an old branch from before that was merged. 

Here it's harmless, but larger changes like that may be annoying to merge.